dist/s_define.list

```diff
@@ -14,7 +14,11 @@ CONNECTION_CONTROL_POINT_DEFINE_RET
 CONNECTION_CONTROL_POINT_DEFINE_SLEEP
 CONNECTION_CONTROL_POINT_DEFINE_START
 CONNECTION_CONTROL_POINT_DEFINE_WAIT_FOR_TRIGGER
```

Comment:
I'm still not a huge fan of this naming system, I expect it to change but in the meantime can we rename this to:
`CONNECTION_CONTROL_POINT_TRIGGER`
`CONNECTION_CONTROL_POINT_WAIT`

I feel like the pairing is more obvious that way?

Reply:
OK, I will rename the action short name from "Wait for trigger" to "Trigger".  The longer part after it still explains it.  You are right this makes the pairing more obvious.